### ISSUE TITLE
consider fwupd support

### ISSUE THREAD

**Issue by bam80** [MrChromebox/firmware#603]
I didn't look at the fwupd closely, but it seems reasonable alternative to the custom script, at least after the alternative fw was initially flashed.

BTW, Coreboot has test account on the LVFS:
https://fwupd.org/lvfs/vendors/

**Comment by bam80** [comment]
As a benefit, it also supports downgrade in case some debugging/bisecting is needed:
https://lvfs.readthedocs.io/en/latest/chromeos.html#downgrade-the-fw-version

**Comment by MrChromebox** [comment]
yes, this is on my to-do list, but adding support for ~300 boards is no small task

**Comment by bam80** [comment]
Any update?

**Comment by MrChromebox** [comment]
nope, haven't had time to look into it

**Comment by bam80** [comment]
Do you need help maybe?

**Comment by MrChromebox** [comment]
won't say no if someone wants to get things started

**Comment by bam80** [comment]
To push things forward, I opened the question upstream:
https://github.com/fwupd/fwupd/issues/9828

**Comment by MrChromebox** [comment]
> To push things forward, I opened the question upstream: [fwupd/fwupd#9828](https://github.com/fwupd/fwupd/issues/9828)

> which is clunky, insecure and not the best way of doing it.

tell me how you really feel, sheesh.

> Could you advice us how to get started, please?

I don't need help getting started, I know how to do it. The logistics and workload to do it for 250+ boards is the problem

**Comment by bam80** [comment]
Sorry, did I give the reason to be rude?
There is no obligation to make it for all the boards at once, IMO, so let's do it at least for one board first.

**Comment by MrChromebox** [comment]
it's just not at the top of the priority list currently. And if I started now, would need to redo it once UEFI capsule updates are implemented.

**Comment by bam80** [comment]
It looks like UEFI capsule update was implemented and upstreamed for coreboot and EDK II by Dasharo:
https://docs.dasharo.com/projects/capsule-updates/

**Comment by MrChromebox** [comment]
> It looks like UEFI capsule update was implemented and upstreamed for coreboot and EDK II by Dasharo: https://docs.dasharo.com/projects/capsule-updates/

it's not fully complete yet. We're working on implementing and updating this for Starlabs devices, once that's done I'll work on it for MrChromebox.
